File: src/app/models/Track.ts
export class Track {
    constructor() {}

    
    private _trackId : number;
    public get trackId() : number {
        return this._trackId;
    }
    public set trackId(v : number) {
        this._trackId = v;
    }
       
    private _name : string;
    public get name() : string {
        return this._name;
    }
    public set name(v : string) {
        this._name = v;
    }
    
}

File: MusicDiscAPI/Models/Track.cs
using Newtonsoft.Json;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace MusicDiscAPI.Models
{
    public class Track
    {
      [JsonProperty("trackId")]
      public string Id { get; set; }
      [JsonProperty("name")]
      public string Name { get; set; }
    }
}
